### PR TITLE
Feature 2716 existing builds

### LINKED ISSUE #2716: Documentation: Move the Existing Builds page to Read the Docs
## Describe the Task ##
In the 20241002 METplus All Hands Meeting the team decided we would move the [Existing Builds](https://dtcenter.org/community-code/metplus/metplus-6-0-existing-builds) page from the website into Read The Docs.

Please put this information on the [METplus Release Information](https://metplus.readthedocs.io/en/feature_2716_existing_builds/Users_Guide/release-notes.html) page.  Please make it section 2.1 where [METplus Components Release Note Links](https://metplus.readthedocs.io/en/feature_2716_existing_builds/Users_Guide/release-notes.html#metplus-components-release-note-links) currently is (and that section would become 2.2).  But, I would also still like to have an existing_builds.rst file that would then show up at the described location on the [METplus Release Information](https://metplus.readthedocs.io/en/feature_2716_existing_builds/Users_Guide/release-notes.html) page.

### Time Estimate ###
<1 day

### Sub-Issues ###
Consider breaking the task down into sub-issues.
- [ ] *Add a checkbox for each sub-issue here.*

### Relevant Deadlines ###
None, but preferably before the 6.0.0 official release.

### Funding Source ###
2782544 (NOAA BASE)

### Assignee ###
- [x] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Review default **alert** labels
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Milestone and Projects ###
- [x] Select **Milestone** as a **METplus-Wrappers-X.Y.Z** version, **Consider for Next Release**, or **Backlog of Development Ideas**
- [x] For a **METplus-Wrappers-X.Y.Z** version, select the **METplus-Wrappers-X.Y.Z Development** project

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdataio](https://github.com/dtcenter/METdataio/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Task Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/appendixA.html#metplus-components-python-packages) table.
- [ ] For any new datasets, an entry to the [METplus Verification Datasets Guide](https://metplus.readthedocs.io/en/latest/Verification_Datasets/index.html).
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issue
Select: **Milestone** as the next official version
Select: **METplus-Wrappers-X.Y.Z Development** project for development toward the next official release
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

### PR DESCRIPTION
**Pull request by lisagoodrich**
## Pull Request Testing ##

- [X] Describe testing already performed for these changes:</br>
I copied the information from [Existing Builds](https://dtcenter.org/community-code/metplus/metplus-6-0-existing-builds).  Originally I added it to one of the pages.  But it was decided it made more sense to have the Existing Builds as its own document.  From there, the 'include::' feature was used to add it to two pages: [METplus Components Release Note Links](https://metplus.readthedocs.io/en/feature_2716_existing_builds/Users_Guide/release-notes.html#metplus-components-release-note-links) as section2.1.  The second location is also section2.1 [METplus Release Information](https://metplus.readthedocs.io/en/feature_2716_existing_builds/Users_Guide/release-notes.html)
I created [Existing Builds conversion into Read the Docs](https://docs.google.com/document/d/1JEJNO0qo4zO6SvGxmJaxd5xtLVJF9ckEMhsgS77vKhI/edit?tab=t.0) to track any difficulties or questions that I had.  Julie has reviewed this page.  As a summary:

1. ReadTheDocs does not bold web pages (unlike the Drupal site)
2. Some of the spacing between the bullet points is inconsistent.  This is a general problem with ReadTheDocs.
3. Any reference to "you" has been changed to third person, "the user" to match with the METplus documentation.
4. "Last Updated" was added to each dropdown menu for consistency.
5. I had difficulty with the "Warning" section under the DERECHO dropdown menu.  I was unable to break the line up to get it under 79 characters without getting a warning message from github.  I don't think anything can be done about this.
6. Under the Docker Hub dropdown menu language was added for the web addresses.

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

1. Please review the original [Existing Builds](https://dtcenter.org/community-code/metplus/metplus-6-0-existing-builds) documentation to the new version that can be accessed from either: [METplus Components Release Note Links](https://metplus.readthedocs.io/en/feature_2716_existing_builds/Users_Guide/release-notes.html#metplus-components-release-note-links) as section2.1 or [METplus Release Information](https://metplus.readthedocs.io/en/feature_2716_existing_builds/Users_Guide/release-notes.html)
2. Please check for bolding or bullet inconsistencies.
3. Please add a date for "Last Updated" if there isn't a date.

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes]**

- [x] Do these changes include sufficient testing updates? **[Yes]**

- [x] Will this PR result in changes to the test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [x] Do these changes introduce new SonarQube findings? **[No]**</br>
If **yes**, please describe:

- [x] Please complete this pull request review by **[10/30/24]**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/appendixA.html#metplus-components-python-packages) table.
- [ ] For any new datasets, an entry to the [METplus Verification Datasets Guide](https://metplus.readthedocs.io/en/latest/Verification_Datasets/index.html).
- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR definition above.
- [ ] Ensure the PR title matches the feature or bugfix branch name.
- [ ] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issue
Select: **Milestone** as the version that will include these changes
Select: **Coordinated METplus-X.Y Support** project for bugfix releases or **METplus-Wrappers-X.Y.Z Development** project for official releases
- [ ] After submitting the PR, select the :gear: icon in the **Development** section of the right hand sidebar. Search for the issue that this PR will close and select it, if it is not already selected.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

**Comment by jprestop** [comment]
Hi @lisagoodrich.  Apologies that I was not more clear - Could you please remove the include from this location in the [Release Guide](https://metplus.readthedocs.io/en/feature_2716_existing_builds/Release_Guide/index.html)?  We only want it to be present in the User's Guide.

File: docs/Users_Guide/existing_builds.rst [hunk 1]
```diff
@@ -0,0 +1,561 @@
...
+forum if no instructions exist for the current release on a supported 
+platform and the release is needed on that platform.
+
+Select from the list below for instructions on using existing builds of 
+the METplus components' software packages. Please note that the commands to 
+load the METplus components assume the user is using bash. If an installation 
+is needed on a machine not listed here, please follow the 
+`installation instructions <https://metplus.readthedocs.io/projects/met/en/latest/Users_Guide/installation.html#software-installation-getting-started>`_ 
+in the 
+`MET User’s Guide <https://metplus.readthedocs.io/projects/met/en/latest/Users_Guide/>`_.
```
Comment (jprestop) [inline docs/Users_Guide/existing_builds.rst]:
```suggestion
`MET User’s Guide <https://met.readthedocs.io/en/latest>`_.
```
The current link you have is correct.  It is a new format where MET is listed as a sub project of METplus.  We haven't decided whether or not we want to use that or not, so I'd like to revert to the old URL, which still works, at this time.

File: docs/Users_Guide/existing_builds.rst [hunk 2]
```diff
@@ -0,0 +1,561 @@
...
+Please submit a new discussion in the 
+`METplus Components Discussion <https://github.com/dtcenter/METplus/discussions>`_
+forum if no instructions exist for the current release on a supported 
+platform and the release is needed on that platform.
+
+Select from the list below for instructions on using existing builds of 
+the METplus components' software packages. Please note that the commands to 
+load the METplus components assume the user is using bash. If an installation 
+is needed on a machine not listed here, please follow the 
+`installation instructions <https://metplus.readthedocs.io/projects/met/en/latest/Users_Guide/installation.html#software-installation-getting-started>`_
```
Comment (jprestop) [inline docs/Users_Guide/existing_builds.rst]:
```suggestion
`installation instructions <https://met.readthedocs.io/en/latest/Users_Guide/installation.html>`_ 
```
The current link you have is correct. It is a new format where MET is listed as a sub project of METplus. We haven't decided whether or not we want to use that or not, so I'd like to revert to the old URL, which still works, at this time.

**Comment by lisagoodrich** [comment]
@jprestop I have made the changes you requested.  Please let me know if there are more changes.

**Review by jprestop — APPROVED** [review]
Thank you so much for all of your work on this task @lisagoodrich.  I appreciate it very much.  I approve.  Please go ahead and squash and merge, delete your branch, and close the issue when you have time.

**Pull request merged**